packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -230,6 +230,7 @@ const usePricingMetaForGridPlans = ( {
 					billingPeriod: plans.data?.[ planSlug ]?.pricing?.billPeriod,
 					// TODO clk: the condition on `.pricing` here needs investigation. There should be a pricing object for all returned API plans.
 					currencyCode: plans.data?.[ planSlug ]?.pricing?.currencyCode,
+					purchaseCurrencyCode: purchasedPlan?.currencyCode,
```

Comment:
@allilevine I feel this may not be the best way to accomplish this. In the same way that we don't provide a separate "purchasedPrice" for the current plan, we probably shouldn't be doing it for currency. I would instead consider updating the single `currency` property to reflect this, because it is currently technically **incorrect** apparently. 

Do you think you can follow up on this?

Reply:
@chriskmnds Thanks for the feedback! I opened a follow-up issue for /plans where we could address this: https://github.com/Automattic/dotcom-forge/issues/7631

Reply:
Thanks! :-)